src/planner/linux/multi.rs

```diff
@@ -36,6 +38,20 @@ impl Planner for LinuxMulti {
             return Err(PlannerError::NixOs);
         }
 
+        // We currently do not support SELinux
+        match Command::new("getenforce").output().await {
+            Ok(output) => {
+                let stdout_string = String::from_utf8(output.stdout).map_err(PlannerError::Utf8)?;
+                tracing::trace!(getenforce_stdout = stdout_string, "SELinux detected");
+                match stdout_string.trim() {
+                    "Enforcing" => return Err(PlannerError::SelinuxEnforcing),
+                    _ => (),
+                }
+            },
+            Err(e) if e.kind() == std::io::ErrorKind::NotFound => (),
+            Err(e) => panic!("{:?}", e),
```

Comment:
Should this really be a panic? Would it be better if it was a `PlannerError::Custom` (of a boxed string or something)?

Reply:
Oh whoops this was for debugging